nf_core/modules/lint/module_version.py

```diff
@@ -25,17 +25,18 @@ def module_version(module_lint_object, module):
 
     # Verify that a git_sha exists in the `modules.json` file for this module
     try:
-        module_entry = module_lint_object.modules_json["modules"][module.module_name]
+        module_entry = module_lint_object.modules_json["repos"]["nf-core/modules"][module.module_name]
```

Comment:
```suggestion
        module_entry = module_lint_object.modules_json["repos"][module_lint_object.modules_repo.name][module.module_name]
```

This is maybe more general/future proof